pkg/kn/commands/route/route_describe.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"errors"
+
+	"github.com/knative/client/pkg/kn/commands"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func NewRouteDescribeCommand(p *commands.KnParams) *cobra.Command {
+	routeDescribePrintFlags := genericclioptions.NewPrintFlags("").WithDefaultOutput("yaml")
+	routeDescribeCommand := &cobra.Command{
+		Use:   "describe NAME",
+		Short: "Describe available route.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return errors.New("requires the route name.")
+			}
+			client, err := p.ServingFactory()
+			if err != nil {
+				return err
+			}
+
+			namespace, err := p.GetNamespace(cmd)
+			if err != nil {
+				return err
+			}
+			describeRoute, err := client.Routes(namespace).Get(args[0], v1.GetOptions{})
+			if err != nil {
+				return err
+			}
+
+			printer, err := routeDescribePrintFlags.ToPrinter()
+			if err != nil {
+				return err
+			}
+			describeRoute.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
```

Comment:
Can we please get #134 merged first as this directly is changed in this PR ?

I mark this PR as 'on-hold' if this ok, and remove it if we have #134 in place. 'hope this is ok.